packages/expo-image/ios/ImageView.swift

```diff
@@ -210,14 +210,17 @@ public final class ImageView: ExpoView {
       log.debug("Loading the image has been canceled")
       return
     }
-    if let image = image {
+
+    // Create an SDAnimatedImage if needed then handle the image
+    if let image = createAnimatedIfNeeded(image: image, data: data) {
```

Comment:
Here we either return an SDAnimatedImage or UIImage depending on whether the image is animated.